pkg/pool-weighted/test/ManagedPool.test.ts

```diff
@@ -869,8 +861,94 @@ describe('ManagedPool', function () {
           it('setting the management fee emits an event', async () => {
             const receipt = await pool.setManagementSwapFeePercentage(owner, NEW_MANAGEMENT_SWAP_FEE_PERCENTAGE);
 
-            expectEvent.inReceipt(await receipt.wait(), 'ManagementFeePercentageChanged', {
-              managementFeePercentage: NEW_MANAGEMENT_SWAP_FEE_PERCENTAGE,
+            expectEvent.inReceipt(await receipt.wait(), 'ManagementSwapFeePercentageChanged', {
+              managementSwapFeePercentage: NEW_MANAGEMENT_SWAP_FEE_PERCENTAGE,
+            });
+
+            it('cannot be set above the maximum AUM fee', async () => {
+              await expect(pool.setManagementAumFeePercentage(owner, fp(0.2))).to.be.revertedWith(
+                'MAX_MANAGEMENT_AUM_FEE_PERCENTAGE'
+              );
+            });
+          });
+        });
+      });
+
+      describe('management aum fee collection', () => {
```

Comment:
We should add tests for 
 - manual collection before initialization
 - manual collection when the pool is paused
 - automatic collection when the pool is paused (proportional exits)

Reply:
Addressed over commits [9b9aebb](https://github.com/balancer-labs/balancer-v2-monorepo/pull/1164/commits/9b9aebba1ef5c17cc11d4d3d2bfa04f04f77db5e) through [0a76238](https://github.com/balancer-labs/balancer-v2-monorepo/pull/1164/commits/0a7623805dccf285cb01ef8016d2c117ad9b9d58)